Automattic/sensei
# Add Course Created email

Resolves #6483 

## Proposed Changes
* Add Course Created email

## Testing Instructions
1. Enable 'email_customization' feature flag.
2. Re-create emails (Sensei LMS -> Tools).
3. Add a teacher.
4. As a teacher, create a course and submit it to review.
5. Check admin's email.

## Screenshots
<img width="1388" alt="CleanShot 2023-03-07 at 16 29 03@2x" src="https://user-images.githubusercontent.com/329356/223382123-8b6f9662-d1b2-4456-8e5f-c33c0a69f2a7.png">

## Pre-Merge Checklist
